lesson3/_homework/task_01/script.js

```diff
@@ -0,0 +1,127 @@
+
+//                    Adapt Academy 2019 lesson 03 - JS introduction | task 01 by Donatas Dereškevičius                     
+
+
+const getZodiacSign = (month, day) => {
+
+    month = document.querySelector(".form-input__month").value;
+    day = document.querySelector(".form-input__day").value;
+
+    if  ((month > 0 && month <= 12) && (day > 0 && day <= 31)){
```

Comment:
But you know, not every moth has 31 days. It might not fail if you provide February 31 even though it has 28 days. But thats a minor issue :)